@AH-64D Apache Longbow/Addons/fza_ah64_controls/config/UserActionGroups.hpp

```diff
@@ -25,7 +25,6 @@ class UserActionGroups
             fza_ah64_wasGun,
             fza_ah64_wasRkt,
             fza_ah64_wasMsl,
-            fza_ah64_masterSafe,
```

Comment:
Do you need to re-add `fza_ah64_armSafeButton`?

Reply:
Keybind is no longer needed